gobblin-modules/gobblin-kafka-common/src/main/java/org/apache/gobblin/kafka/writer/KafkaWriterHelper.java

```diff
@@ -90,4 +94,32 @@ public static Object getKafkaProducer(Properties props) {
       throw Throwables.propagate(e);
     }
   }
+
+  public static <K, D> K getKey(D record, KafkaWriterCommonConfig commonConfig)
+      throws FieldMappingException {
+    if (commonConfig.isKeyed()) {
+      return (K) commonConfig.getTypeMapper().getField(record, commonConfig.getKeyField());
+    } else {
+      return null;
+    }
+  }
+
+  public static <D> D getValue(D record, KafkaWriterCommonConfig commonConfig)
+      throws FieldMappingException {
+    return (D) commonConfig.getTypeMapper().getField(record, commonConfig.getValueField());
+  }
+
+  public static <V, K> Pair<K,V> getKeyValuePair(V record, KafkaWriterCommonConfig commonConfig)
+      throws FieldMappingException {
+    K key = null;
+    TypeMapper typeMapper = commonConfig.getTypeMapper();
```

Comment:
Why not call `getKey` and `getValue` to avoid code duplication? Is this to be a little more efficient?

Reply:
Will remove the getKey and getValue methods. Not using them anymore.